generator/schemas/gen.go

```diff
@@ -364,7 +395,13 @@ func addMarkdownDescription(orderedMap *orderedmap.OrderedMap) {
 	}
 	description, descriptionExists := orderedMap.Get("description")
 	if descriptionExists {
-		orderedMap.Set("markdownDescription", description)
+		s := fmt.Sprintf("%v", description)
+		if strings.HasPrefix(s, "Map of implementation-dependant") {
```

Comment:
Am I missing something, or the same thing is done in both conditions  ?

Reply:
:facepalm: it should be reverted, I crafted it for debugging. Good catch!

Reply:
reverted